### PR TITLE
use `instance_eval` instead of `module_eval`

### PR DESCRIPTION
**Pull request by kibitan**
related with #273 

`rails runner` execute the file with `Rails` module in rails 3.*, 4.0 and the difference with `module_eval` and `instance_eval` is context of evaling code.
- module_eval
  - it will execute with `eigenclass` of `self` context. inside of eval code `def` will define class method.
- instance_eval
  - it will execute with `class` of `self` context. inside of eval code `def` will define instance method.

### CODE REVIEW

File: spec/bin/rollbar_runner_spec.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  end
+end
+
+describe Rails::RollbarRunner do
+  context 'with file contains define method in top level' do
+    # refs: https://github.com/rollbar/rollbar-gem/issues/273
+
+    before do
+      with_dummyapp_dir do
+        File.open('./lib/test_ruunner.rb', 'w') do |file|
```
Comment (jondeandres) [inline spec/bin/rollbar_runner_spec.rb]: It'd be nice having this file written in the dummyapp instead of write it for every test.

**Comment by jondeandres** [comment]
Hey @ctokoro, thanks for this!

Probably it has more sense use `#instance_eval` instead of `#module_eval`, yes. I'm looking there are several builds failing in travis.

* Some of them are failing cause the output is not the expected, https://travis-ci.org/rollbar/rollbar-gem/jobs/76092944.
* Others fail cause `byebug` gem is not found, https://travis-ci.org/rollbar/rollbar-gem/jobs/76092939. I think this is happening cause when you execute `rollbar-rails-runner ./lib/test_ruunner.rb` maybe the environment is not used and the `Gemfile` in the repo root is being used. Instead of the defined by `BUNDLE_GEMFILE`.
  * Another option is that you should execute that command with `bundle rollbar-rails-runner`.

**Comment by ctokoro** [comment]
@jondeandres 
thanks your review and kindly advice!
i will try to fix it :smile:

**Comment by brianr** [comment]
@ctokoro Do you think you'll get to this relatively soon?

**Comment by ctokoro** [comment]
@brianr CC: @jondeandres 
Actually not yet. And I've found another problem with using `#instance_eval` currently.
with `include module` in runner script, module will be not included well.. (`NoMethodError` is happened.)
I've not research definitely reproduce conditions of it yet, but now I think this PR is not good solution of #273 .

**Comment by coryvirok** [comment]
@rokob can you take a look and see if there's a good solution?

**Comment by rokob** [comment]
Related to some other rails-runner issues #551, will take a look soon

**Comment by rivkahstandig3636** [comment]
@ArturMoczulski Can you review this PR?

**Comment by ArturMoczulski** [comment]
Issue #273 has been closed and this PR is conflicting with the current master. If this is still a requested feature I will reopen.

**Pull request closed without merging**